packages/pharos/src/components/coach-mark/pharos-coach-mark.scss

```diff
@@ -37,24 +37,20 @@
 }
 
 :host {
-  --coach-mark-background-color: var(--pharos-color-white);
-  --coach-mark-border-color: var(--pharos-color-marble-gray-base);
-  --coach-mark-border-width: 1px;
-  --coach-mark-text-size: var(--pharos-font-size-small);
-  --coach-mark-heading-color: var(--pharos-color-marble-gray-20);
-  --coach-mark-text-color: var(--pharos-color-marble-gray-20);
-  --coach-mark-text-spacing: var(--pharos-spacing-1-x);
-  --coach-mark-pip-size: 10px;
+  --coach-mark-background-color: var(--pharos-coach-mark-color-background-light);
+  --coach-mark-border-color: var(--pharos-coach-mark-color-border-light);
+  --coach-mark-heading-color: var(--pharos-coach-mark-color-heading-light);
+  --coach-mark-text-color: var(--pharos-coach-mark-color-text-light);
 
   display: block;
   position: absolute;
 }
 
 :host([variant='dark']) {
-  --coach-mark-background-color: var(--pharos-button-color-base-overlay-background-base);
-  --coach-mark-border-color: var(--coach-mark-background-color);
-  --coach-mark-heading-color: var(--pharos-color-white);
-  --coach-mark-text-color: var(--pharos-color-marble-gray-80);
+  --coach-mark-background-color: var(--pharos-coach-mark-color-background-dark);
+  --coach-mark-border-color: var(--pharos-coach-mark-color-border-dark);
+  --coach-mark-heading-color: var(--pharos-coach-mark-color-heading-dark);
+  --coach-mark-text-color: var(--pharos-coach-mark-color-text-dark);
```

Comment:
Sorry, it seems I wasn't too clear before; I think you will want to remove these custom properties now, and have CSS below like the following:

```css
... {
  background: var(--pharos-coach-mark-background-color, var(/* the raw value from before */));
}
```